27-04-2023 07:37 ERROR TelegramBotListener:256 - Ответ не был отправлен
java.lang.RuntimeException
	at re.st.animalshelter.TelegramBotListener.send(TelegramBotListener.java:256)
	at re.st.animalshelter.TelegramBotListener.sendEditedTextResponse(TelegramBotListener.java:238)
	at re.st.animalshelter.TelegramBotListener.processBack(TelegramBotListener.java:151)
	at re.st.animalshelter.TelegramBotListener.processCallbackQuery(TelegramBotListener.java:105)
	at re.st.animalshelter.TelegramBotListener.process(TelegramBotListener.java:78)
	at com.pengrad.telegrambot.impl.UpdatesHandler$1.onResponse(UpdatesHandler.java:73)
	at com.pengrad.telegrambot.impl.UpdatesHandler$1.onResponse(UpdatesHandler.java:49)
	at com.pengrad.telegrambot.impl.TelegramBotClient$1.onResponse(TelegramBotClient.java:53)
	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:519)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
